server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java

```diff
@@ -119,6 +121,9 @@ void addFilePathToMap(long queryId, TsFileResource tsFile, boolean isClosed) {
     if (!pathMap.get(queryId).contains(tsFile)) {
       pathMap.get(queryId).add(tsFile);
       FileReaderManager.getInstance().increaseFileReaderReference(tsFile, isClosed);
+      logger.warn("{} is read-locked by {}", tsFile, queryId, new Exception());

```

Comment:
Is this a debug log? Why print the exception?

Reply:
Removed.